[NEW] Realtime InAppChat using RocketChat-SDK

sidmohanty11 opened the pull request:
- Uses the `rockechat-sdk` to make inappchat realtime
- Separated helper functions to avoid cluttering in `Inappchat.js`
- I've added TODO comment where the animation should occur
- [x]  Separated emojify logic where, a TODO is there to import Emojione (as an npm module) and use it instead of <Script />, which @demonicirfan has worked on his upcoming progress with animations.  (DONE IN THIS PR)

- Now we only need the URL/HOST ENV variable at prod

## Preview

https://user-images.githubusercontent.com/73601258/157447703-0c732fb9-35d0-468c-bc1d-1d236e474539.mp4

Sing-Li reviewed (changes requested):
please remove pacakage-lock.json for the PR.  Thanks.

Sing-Li commented:
Please resolve conflicts as well  @sidmohanty11    Thanks.

sidmohanty11 commented:
@Sing-Li I don't know how it got committed in the first place, maybe it's my vs-code but I've removed the `package-lock` file, updated the docs a bit (of inappchat) and added the emoji's npm module and removed all the script tags.

Now, using Inappchat is a breeze as, we just need to do this and not worry about anything else

```jsx
<InAppChat closeChat={handleOpenChat} cookies={cookies} rid={rid} />
```
